versioned_docs/version-v4.12/components/api.md

```diff
@@ -81,20 +81,104 @@ window.addEventListener('appload', (event) => {
 
 ## Other
 
-- [**Host**](./host-element.md): Host is a functional component that can be used at the root of the render function to set attributes and event listeners to the host element itself.
+The following primitives can be imported from the `@stencil/core` package and used within the lifecycle of a component:
+
+- [**Host**](./host-element.md): `<Host>`, is a functional component that can be used at the root of the render function to set attributes and event listeners to the host element itself. Refer to the [Host Element](./host-element.md) page for usage info.
+
+- **Fragment**: `<Fragment>`, often used via `<>...</>` syntax, lets you group elements without a wrapper node. Make sure you define a `jsxFragmentFactory` compiler option alongside `jsxFactory` when using this functional component.
```

Comment:
Can we clarify which compiler (stencil vs typescript) this is for? 

I tried to add the word "TypeScript" along with the value setting to "Fragment" and my edits didn't sound good in my head. What do you think about:

```suggestion
- **Fragment**: `<Fragment>`, often used via `<>...</>` syntax, lets you group elements without a wrapper node.

To use this feature, ensure that the following TypeScript compiler options are set:
- [`jsxFragmentFactory` is set](https://www.typescriptlang.org/tsconfig#jsxFragmentFactory) to "Fragment"
- [`jsxFactory` is set](https://www.typescriptlang.org/tsconfig#jsxFactory) to "h"
```